scripts/msg.h.template

```diff
@@ -98,6 +117,12 @@ struct @(spec.short_name)_
   typedef boost::shared_ptr< @(cpp_full_name)<ContainerAllocator> > Ptr;
   typedef boost::shared_ptr< @(cpp_full_name)<ContainerAllocator> const> ConstPtr;
 
+@# Plugins
+@[if have_plugin]@
+#ifdef @(spec.package.upper())_MESSAGE_@(spec.short_name.upper())_PLUGIN
+  @(spec.package.upper())_MESSAGE_@(spec.short_name.upper())_PLUGIN
```

Comment:
I think this macro name should contain something in the name which clarifies that it is "additional stuff in the class body".
Together with the feedback from another comment below maybe: `@(spec.package.upper())_MESSAGE_@(spec.short_name.upper())_PLUGIN_CLASS_BODY`.